doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -492,13 +509,15 @@ When you have all the source files that you need, run the configure script, whic
 
 ```
 bash configure --with-freemarker-jar=/<my_home_dir>/freemarker.jar \
+               --with-freetype=/usr/local/Cellar/freetype/2.9.1/ \
```

Comment:
`--with-freetype` is not required by default. it should be included as an optional step similar to **Non-compressed references support**.

Reply:
For clarity, is freetype a definite pre-req? If not, in what situation would a user need to include the --with-freetype option? Answers will help me phrase the note. Thanks

Reply:
Yes, `freetype` a definite pre-req. `--with-freetype` can be used in situations where `bash configure` is unable to auto-detect `freetype`.

Reply:
Thanks. Note added.